src/pages/Movie/index.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Container } from 'react-bootstrap';
+
+class Movie extends React.Component {
+  state = {
+    title: "in progress...",
```

Comment:
Let's add only empty strings for default, in the next sprints we will implement some loader logic for this